src/core/templates/core/password.html

```diff
@@ -49,4 +53,49 @@ <h4 style="color: red">{{password_msg}}</h4>
       </form>
     </div>
   </div>
+  <form id="enable_2fa_form">
+  <div class="modal fade" id="enable_2fa_modal" data-backdrop="static" data-keyboard="false">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" id="enable_close_btn" class="close" aria-hidden="true">&times;</button>
+          <h4 class="modal-title">{% trans 'Setup Authenticator App' %}</h4>
+        </div>
+        <div class="modal-body">
+          <p>{% trans 'Please set up the QR code with the authentication application.' %}</p>
+          <img id="qrcode_img" width="225" height="225" alt="qrcode">
+          <p id="secret"></p>
+          <p>{% trans 'Input authentication code (6 digits)' %}</p>
+          {% csrf_token %}
+          <input type="text" id="authentication_code" name="authentication_code" maxlength="6" required autocomplete="off"></inputs>
+          <p class="err_msg"></p>
+        </div>
+        <div class="modal-footer">
+          <button type="button" id="enable_cancel_btn" class="btn btn-default">{% trans 'Cancel' %}</button>
+          <button type="submit" id="enable_submit_btn" class="btn btn-primary">{% trans 'Submit' %}</button>
+        </div>
+      </div>
+    </div>
+  </div>
+  </form>
+  <form id="disable_2fa_form">
+    <div class="modal fade" id="disable_2fa_modal" data-backdrop="static" data-keyboard="false">
+      <div class="modal-dialog">
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" id="disable_close_btn" class="close" aria-hidden="true">&times;</button>
+            <h4 class="modal-title">{% trans 'Disable 2FA settings?' %}</h4>
```

Comment:
There is no sentence in modal content. It may be strange.
We should set `{% trans 'Disable 2FA settings?' %}` in `modal-body`.
It is good to set "Confirmation (or 確認)" in `modal-header`.

Reply:
@tkhsknsk 

I forgot to ask you to modify the message.

Disable 2FA settings? -> Disable 2FA?